models/mart/current_mrr.sql

```diff
@@ -4,19 +4,68 @@
     )
 }}
 
-SELECT 
-  sim.subscription_id,
-  sim.customer_id,
-  c.name,
-  sum(mrr) as mrr,
-  sum(brl_mrr) as brl_mrr,
-  pc.name as product_class,
-  sim.stripe_account
-FROM {{ref('stripe__subscription_items_mrr')}} sim
-  left join {{source('dbt_stripe_src', 'customer')}} c on c.id = sim.customer_id and sim.stripe_account = c.stripe_account
-  left join {{source('dbt_stripe_src', 'product')}} p on sim.product_id = p.id 
-  left join {{source('dbt_stripe_src', 'product_classes')}} pc on pc.id = p.product_class
-where mrr > 0
-and customer_id NOT IN {{var('exception_ids')}}
-group by 1,2,3,6,7
-order by 2 asc
\ No newline at end of file
+with subscription_item_location as (
+  select
+    *
+  from
+    {{ ref('subscription_item_location') }}
+),
+location_quantity as (
+  select
+    subscription_id,
+    subscription_item_id,
+    sum(device_count) as quantity
+  from
+    subscription_item_location
+  group by 1,2
+  order by 1
+),
+mrr as (
+  select
+    sim.subscription_id,
+    sim.customer_id,
+    c."name",
+    item_info.subscription_item_id,
+    item_info.product_class,
+    p.id as product_id,
+    p.name as product_name,
+    CASE
+      WHEN item_info.product_class != 'Bare Metal' THEN NULL
+      WHEN p.name ILIKE '%g3%' THEN 'Accelerate'

```

Comment:
I think it must start with `g3` right?

```suggestion
      WHEN p.name ILIKE 'g3%' THEN 'Accelerate'

```